common/common.go

```diff
@@ -31,6 +31,7 @@ const (
 	MetadataServiceIP      = "169.254.169.255"
 	MetadataServicePubPort = 80
 	MetadataServicePrvPort = 2375
+	MetadataServiceRegSock = "/var/run/rkt-metadata-svc.sock"
```

Comment:
Well, this is a migration that started in ~2011, and is proposed for FHS; IMHO we should help nudge people into the future, as we do elsewhere ;-)

references:
https://lists.fedoraproject.org/pipermail/devel/2011-March/150031.html
https://wiki.debian.org/ReleaseGoals/RunDirectory
http://askubuntu.com/questions/57297/why-has-var-run-been-migrated-to-run